ext/openssl/tests/ServerClientTestCase.inc

```diff
@@ -119,44 +124,55 @@ class ServerClientTestCase
         eval($code);
     }
 
-    public function run($masterCode, $workerCode)
+    public function run($masterCode, $workerCode, bool $empheral = false)
```

Comment:
I think it would be better for `$ephemeral` (notice the typo) to be true by default (or mandatory) rather than the reverse.
```suggestion
    public function run($masterCode, $workerCode, bool $ephemeral = ture)
```

Reply:
`true` might be better than `ture` :)

Reply:
lol I have no idea why I thought that this is the right spelling of that word...

that default was kept only for migration when I wrote but I managed to do almost all test in one go so I could drop it but will need to change the old ones to use it as well.